src/core_write.cpp

```diff
@@ -145,7 +145,7 @@ void AddRangeproof(const std::vector<uint8_t> &vRangeproof, UniValue &entry)
     {
         int exponent, mantissa;
         CAmount min_value, max_value;
-        if (0 == GetRangeProofInfo(vRangeproof, exponent, mantissa, min_value, max_value))
+        if (GetRangeProofInfo(vRangeproof, exponent, mantissa, min_value, max_value))
```

Comment:
side note.  it's actually one habit I picked up from my indian team.  const == value rather than 'value == constant'.  Everyone who uses 'value == constant' has had a time or 3 of debugging where they had an accidental 'if (value = constant)'.  Since I began adopting 'if (constant == value)', I have had one instance where the compiler caught my 'constant = value' mistake.  Therefore, worth the weirdness :)